src/test/java/org/folio/inventory/validation/experimental/IntellectualItemStatusValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.inventory.validation.experimental;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import lombok.SneakyThrows;
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.domain.items.Status;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.inventory.domain.items.ItemStatusName.INTELLECTUAL_ITEM;
+import static org.folio.inventory.domain.items.ItemStatusName.IN_PROCESS;
+
+@RunWith(JUnitParamsRunner.class)
+public class IntellectualItemStatusValidatorTest {
+  private static TargetItemStatusValidator validator;
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    validator = new TargetItemStatusValidator();
+  }
```

Comment:
Is this field used anymore?

Reply:
Of course. It gives validatators for each test.

Reply:
As far as I could tell, the only test in this class does not use that field.